runtime/gc_realtime/MetronomeAlarm.cpp

```diff
@@ -20,22 +20,9 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-#include "ProcessorInfo.hpp"
-#include "AtomicOperations.hpp"
-#include "ClassModel.hpp"
-#include "Dispatcher.hpp"
 #include "EnvironmentBase.hpp"
-#include "GCExtensions.hpp"
-#include "Heap.hpp"
-#include "MemoryPoolSegregated.hpp"
-#include "MemorySubSpace.hpp"
-#include "modronapi.hpp"
-#include "ObjectModel.hpp"
-#include "OSInterface.hpp"
-#include "RootScanner.hpp"
-#include "Task.hpp"
 #include "MetronomeAlarmThread.hpp"
-#include "RealtimeGC.hpp"
+#include "OSInterface.hpp"
```

Comment:
Looks like we need to add `#include "Scheduler.hpp"` here. It is used in `MM_ITAlarm::alarm_handle()` I don't know how it resolved now.